Sources/SwiftInspectorVisitors/Tests/StructVisitorSpec.swift

```diff
@@ -271,7 +282,7 @@ final class StructVisitorSpec: QuickSpec {
           it("asserts") {
             let content = """
             public struct FooStruct {}
-            public struct FooClass {}
```

Comment:
another copy/paste that I found when duplicating this structure for the class visitor 🤦‍♂️